### DISCUSSION TITLE
Change default page/section

### DISCUSSION THREAD

**Discussion by lesterbotello** [alshedivat/al-folio#2338] [Q&A] [answered]
Hi!

I looked for an answer to this, but I couldn't find it, so opening a new thread (hope it's ok).

The template is working great for me, but I would like to customize it even further and I'm not super familiar with Jekyll. I'm wondering if there's a way to have the site start in a different section by default (for example, "blogs" instead of "about"). What would be the best way to do it?

Thanks in advance.

**Comment by george-gca** [comment, marked as answer]
Check #2305.
